app/api/answers/get_answers.go

```diff
@@ -68,23 +104,43 @@ type rawAnswersData struct {
 }
 
 type answersResponseAnswerUser struct {
-	Login     string  `json:"login"`
+	// required: true
+	Login string `json:"login"`
+	// Nullable
 	FirstName *string `json:"first_name,omitempty"`
-	LastName  *string `json:"last_name,omitempty"`
+	// Nullable
+	LastName *string `json:"last_name,omitempty"`
 }
 
 type answersResponseAnswer struct {
-	ID             int64    `json:"id,string"`
-	Name           *string  `json:"name,omitempty"`
-	Type           string   `json:"type"`
-	LangProg       *string  `json:"lang_prog,omitempty"`
-	SubmissionDate string   `json:"submission_date"`
-	Score          *float32 `json:"score,omitempty"`
-	Validated      *bool    `json:"validated,omitempty"`
-
+	// required: true
+	ID int64 `json:"id,string"`
+	// Nullable
+	Name *string `json:"name,omitempty"`
+	// required: true
+	// enum: Submission,Saved,Current
+	Type string `json:"type"`
+	// Nullable
+	LangProg *string `json:"lang_prog,omitempty"`
+	// required: true
+	SubmissionDate string `json:"submission_date"`
+	// Nullable
+	Score *float32 `json:"score,omitempty"`
+	// Nullable
+	Validated *bool `json:"validated,omitempty"`
```

Comment:
Why is doing already now with this "omitempty"? I don't get it. Should not be omitted.

Reply:
fixed